conversations.yaml

```diff
@@ -0,0 +1,596 @@
+                  is the time of the last message-part.
+              message:
+                properties:
+                  mid:
+                    type: string
+                    format: uuid
+                    description:
+                     An identifier for the message sent. Used for tracking 
+                     and debugging.
+                  text:
+                    type: string
+                    description:
+                      The message text, no longer than 1,600 characters.
+                      The channel service may split the message into
+                      multiple parts for delivery.
+    ConversationTrigger:
+      properties:
+        id:
+          type: string
+          description:
+            The PageId value configured in Medallia Conversations when 
+            the channel is created.
+        time:
+          type: integer
+          format: int64
+          description:
+            Unix epoch timestamp of the last event associated with the
+            request. For example, when it is a multi-part message, this
+            is the time of the last message-part.
+        messaging:
+          type: array
+          items:
+            properties:
+              sender:
+                properties:
+                  id:
+                    type: string
+                    description:
+                      An identifier unique to the customer, such as phone
+                      number, email, or other channel-specific value.
+                      For phone numbers, the ID must be in E.164 format.
+              recipient:
+                properties:
+                  id:
+                    type: string
+                    description:
+                      The PageId value configured in Medallia Conversations 
+                      when the channel is created.
+              timestamp:
+                type: integer
+                format: int64
+              message:
+                properties:
+                  mid:
+                    type: string
+                    format: uuid
+                    description:
+                     An identifier for the message sent. Used for tracking 
+                     and debugging.
+                  text:
+                    type: string
+                    description:
+                      The message text, no longer than 1,600 characters.
+                      The channel service may split the message into
+                      multiple parts for delivery.
+              trigger_parameters:
+                properties:
+                  override_session:
+                    type: boolean
+                  consumer:
+                    properties:
+                      id:
+                        type: string
+                      email:
+                        type: string
+                        format: email
+                  custom:
```

Comment:
Change this to be:

```
custom:
  type: object
  additionalProperties: 
    type: string
```

https://stackoverflow.com/questions/41097913/swagger-complex-response-model-with-dynamic-key-value-hash-maps